2024-01-19 10:28:25 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trading/platform/
2024-01-19 10:28:25 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 82, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 293, in element_is_visible
    return Wait(self.browser, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.11/lib/python3.11/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x562184ec0f83 <unknown>
#1 0x562184b79cf7 <unknown>
#2 0x562184bc999e <unknown>
#3 0x562184bc9aa1 <unknown>
#4 0x562184c14d64 <unknown>
#5 0x562184bf30dd <unknown>
#6 0x562184c12006 <unknown>
#7 0x562184bf2e53 <unknown>
#8 0x562184bbadd4 <unknown>
#9 0x562184bbc1de <unknown>
#10 0x562184e85531 <unknown>
#11 0x562184e89455 <unknown>
#12 0x562184e71f55 <unknown>
#13 0x562184e8a0ef <unknown>
#14 0x562184e5599f <unknown>
#15 0x562184eae008 <unknown>
#16 0x562184eae1d7 <unknown>
#17 0x562184ec0124 <unknown>
#18 0x7fc584c94ac3 <unknown>